src/lib/product_reviews/review/header.js

```diff
@@ -51,7 +45,7 @@ const Header = ({ rating, headline, date, productName, productVariantName }) =>
                {productName} | {productVariantName}
             </ProductVariantName>
          )}
-         <Timestamp itemProp="datePublished">{date}</Timestamp>
+         <Timestamp>{date}</Timestamp>
```

Comment:
This `date` isn't available to us when we are building `schemaReviews` which is why we're not including `datePublished` there, right? Just trying to check my understanding.

Reply:
We have access to it and I can add it in, just wanted to add in necessary data first and adjust as needed